docs/dev-guide.rst

```diff
@@ -206,26 +206,15 @@ Release notes
 =============
 
 To add entries to the release notes, create a file in the ``news`` directory
-with the ``source.type`` name format, where ``type`` is one of:
-
-* ``feature``: for new features
```

Comment:
I don't think it's good idea to remove this text. We should still have list of supported labels.